.github/workflows/ci.yml

```diff
@@ -30,6 +31,7 @@ jobs:
       - run: npm run lint
 
   test_unit:
+    needs: lint
```

Comment:
Why does this depend on the `lint` job? The reason they were split up in the first place was so that they can run in parallel. If we no longer want them to run in parallel, we should just move the `npm run test` step into the `lint` job, right under the lint step, since we won't have to set up the repository twice sequentially.

Reply:
Very good point. I missed this. 100% agree, I think these should stay parallel.

Reply:
Will return to prior